### PR TITLE
fix(android): touchFeedback support for ListView and TableView

### PR DESCRIPTION
**Pull request by garymathews**
- Fix `touchFeedback` support for ListView and TableView

##### TEST CASE 1
- `touchFeedback` functions correctly, with and without `borderRadius`
```JS
const tabGroup = Ti.UI.createTabGroup({
    tabs: [
        Ti.UI.createTab({
            window: createListViewWindow(),
            title: 'ListView'
        }),
        Ti.UI.createTab({
            window: createTableViewWindow(),
            title: 'TableView'
        })
    ]
});
tabGroup.open();

function createListViewWindow() {
    const win = Ti.UI.createWindow({
        backgroundColor: 'white'
    });
    const section = Ti.UI.createListSection({
        items: [{
                properties: {
                    backgroundColor: 'gray'
                },
                title: {
                    text: 'Item #1 - gray',
                    color: 'black'
                }
            },
            {
                properties: {
                    backgroundColor: 'transparent'
                },
                title: {
                    text: 'Item #2 - transparent',
                    color: 'black'
                }
            },
            {
                title: {
                    text: 'Item #3 - undefined',
                    color: 'black'
                }
            },
        ]
    });
    const listView = Ti.UI.createListView({
        sections: [section],
        templates: {
            template: {
                properties: {
                    height: 50,
                    borderRadius: 10 // QE: Test without borderRadius
                },
                childTemplates: [{
                    type: 'Ti.UI.Label',
                    bindId: 'title',
                    properties: {
                        left: '5dp',
                        color: 'white'
                    }
                }]
            }
        },
        defaultItemTemplate: 'template',
        touchFeedback: true,
        touchFeedbackColor: 'green'
    });
    const isColor = _ => `touchFeedbackColor: ${listView.touchFeedbackColor == 'red' ? 'red' : 'blue'}`;
    const btn_color = Ti.UI.createButton({
        title: isColor(),
        bottom: 5,
        right: 5
    });
    btn_color.addEventListener('click', _ => {
        listView.touchFeedbackColor = listView.touchFeedbackColor == 'red' ? 'blue' : 'red';
        btn_color.title = isColor();
    });
    const isFeedback = _ => `touchFeedback: ${listView.touchFeedback ? 'ON' : 'OFF'}`;
    const btn_feedback = Ti.UI.createButton({
        title: isFeedback(),
        bottom: 5,
        left: 5
    });
    btn_feedback.addEventListener('click', _ => {
        listView.touchFeedback = !listView.touchFeedback;
        btn_feedback.title = isFeedback();
    });
    win.add([listView, btn_feedback, btn_color]);
    return win;
}

function createTableViewWindow() {
    const win = Ti.UI.createWindow({
        backgroundColor: 'white'
    });
    const tableView = Ti.UI.createTableView({
        backgroundColor: 'white',
        data: [{
                title: 'Item #1',
                color: 'white',
                backgroundColor: 'gray',
                borderRadius: 10, // QE: Test without borderRadius
                height: 50
            },
            {
                title: 'Item #2 - transparent',
                color: 'black',
                backgroundColor: 'transparent',
                borderRadius: 10,
                height: 50
            },
            {
                title: 'Item #3 - undefined',
                color: 'black',
                borderRadius: 10,
                height: 50
            }
        ],
        touchFeedback: true,
        touchFeedbackColor: 'green'
    });
    const isColor = _ => `touchFeedbackColor: ${tableView.touchFeedbackColor == 'red' ? 'red' : 'blue'}`;
    const btn_color = Ti.UI.createButton({
        title: isColor(),
        bottom: 5,
        right: 5
    });
    btn_color.addEventListener('click', _ => {
        tableView.touchFeedbackColor = tableView.touchFeedbackColor == 'red' ? 'blue' : 'red';
        btn_color.title = isColor();
    });
    const isFeedback = _ => `touchFeedback: ${tableView.touchFeedback ? 'ON' : 'OFF'}`;
    const btn_feedback = Ti.UI.createButton({
        title: isFeedback(),
        bottom: 5,
        left: 5
    });
    btn_feedback.addEventListener('click', _ => {
        tableView.touchFeedback = !tableView.touchFeedback;
        btn_feedback.title = isFeedback();
    });
    win.add([tableView, btn_feedback, btn_color]);
    return win;
}
```

##### TEST CASE 2
- Moving a row with no background uses parents `backgroundColor`
```JS
const win = Ti.UI.createWindow();
const tableView = Ti.UI.createTableView({
        // backgroundColor: 'red',
	data: [
		{
			title: 'Item #1 (editable: true)',
			editable: true,
		},
		{
			title: 'Item #2 (movable: true)',
			movable: true,
		},
		{
			title: 'Item #3 (default)',
		}
	]
});
const isEditing = () => `Editing: ${tableView.editing ? 'ON' : 'OFF'}`;
const btn_editing = Ti.UI.createButton({
	title: isEditing(),
	bottom: 5,
	right: 5
});
btn_editing.addEventListener('click', () => {
	tableView.editing = !tableView.editing;
	btn_editing.title = isEditing();
});
const isMoving = () => `Moving: ${tableView.moving ? 'ON' : 'OFF'}`;
const btn_moving = Ti.UI.createButton({
	title: isMoving(),
	bottom: 5,
	left: 5
});
btn_moving.addEventListener('click', () => {
	tableView.moving = !tableView.moving;
	btn_moving.title = isMoving();
});
win.add([tableView, btn_editing, btn_moving]);
win.open();
```

[TIMOB-28329](https://jira.appcelerator.org/browse/TIMOB-28329)
[TIMOB-27504](https://jira.appcelerator.org/browse/TIMOB-27504)

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java [hunk 1]
```diff
@@ -65,6 +67,7 @@ public ListViewProxy()
 		defaultValues.put(TiC.PROPERTY_CASE_INSENSITIVE_SEARCH, true);
 		defaultValues.put(TiC.PROPERTY_DEFAULT_ITEM_TEMPLATE, UIModule.LIST_ITEM_TEMPLATE_DEFAULT);
 		defaultValues.put(TiC.PROPERTY_FAST_SCROLL, false);
+		defaultValues.put(TiC.PROPERTY_TOUCH_FEEDBACK, false);
```
Comment (garymathews) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java:70]: I have implemented this to be `false` to follow our docs. But we may want to change `touchFeedback` to default `true` instead.

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Review by ssjsamir — APPROVED** [review]
FR Passed, tested using the test cases mentioned above. 

**Test Environment**
```
MacOS Big Sur: 11.1 Beta 1
Xcode: 12.2 Beta
Java Version: 1.8.0_242
Android NDK: 21.3.6528147
Node.js: 12.18.1
""NPM":"5.0.0","CLI":"8.1.1""
iPhone 8 14.2
```

**Comment by sgtcoolguy** [comment]
manually cherry-picked the squashed commit to `10_0_X`

**Pull request merged**